lms/djangoapps/courseware/url_helpers.py

```diff
@@ -42,11 +42,24 @@ def get_redirect_url(course_key, usage_key):
         # Here we use the navigation_index from the position returned from
         # path_to_location - we can only navigate to the topmost vertical at the
         # moment
-
         redirect_url = reverse(
             'courseware_position',
             args=(unicode(course_key), chapter, section, navigation_index(position))
         )
-
     redirect_url += "?{}".format(urlencode({'activate_block_id': unicode(final_target_id)}))
     return redirect_url
+
+
+def get_redirect_url_for_global_staff(course_key, location):
+    """
+    Returns the redirect url
```

Comment:
maybe something more clear? `Returns the redirect url for staff enrollment` or whatever makes it more clear.